### PR TITLE
optimize images in PastMemories, students, and projects dirs

### PR DESCRIPTION
**Pull request by ammarbinfaisal**
- [x] Included a Preview link and screenshot showing after and before the changes.
- [x] Included a description of the change below.
- [x] Squashed the commits.

# Changes done in this Pull Request

- [Preview Link](https://ammarbinfaisal.github.io/gci18.fossasia.org)
- for #542 and #541
- @Sid911 did the conversions in mentors direcctory
- I've used @Sid911 's system of fallback on jpg/png images if webp dont load

## Description / Changes
- in PAstMemories, students and mentors directory
- **encoded jpgs using mozjpeg encoder**
-  **and converted jpgs/pngs to webps (unless the image was too small)**

- - - - - - - - - - - -

### CODE REVIEW

File: _data/mentors.yml [hunk 1]
```diff
@@ -56,58 +66,70 @@
 
 - name: Mohit Sharma
   image: ms.jpg
+  image: ms.webp
```
Comment (jogendra) [inline _data/mentors.yml]: change it to `imagewebp`

**Comment by jogendra** [comment]
@ammarbinfaisal please resolve conflicts

**Comment by ammarbinfaisal** [comment]
@abishekvashok okay

**Comment by ammarbinfaisal** [comment]
sorry, I ddnt remember that branch was there for this pull request! Git branching confuses me. I am sorry

**Comment by ammarbinfaisal** [comment]
@jogendra @abishekvashok please have a look here agin

**Pull request merged**